spark/README.md

```diff
@@ -0,0 +1,5 @@
+# code-examples
+
+2 examples to test understanding of code/scala/spark
+
+SnowplowHelper is easier and smaller than BidResponseJob, but it is one's own choice which one to start with. 
```

Comment:
"to start with": Does that mean that the candidate should go through both of them to get hired? In case yes, I think we can do without this comment